pattern-1/apim/wso2apim-deployment.yaml

```diff
@@ -79,11 +79,11 @@ spec:
           protocol: "TCP"
```

Comment:
IMO, we should define the resource requirements and limits for API Manager based on the recommendations [1].

WDYT?

[1]: [Resource requirements and limits](https://github.com/wso2/kubernetes-apim/blob/2.2.x/pattern-1/apim/wso2apim-deployment.yaml#L56)

Reply:
Yes, this is a good idea. Let's try to add this if time permits.